bedrock/mozorg/redirects.py

```diff
@@ -588,6 +588,11 @@ def decider(request, **kwargs):
     # bug 1003737
     redirect(r'^impressum/?$', '/about/legal/impressum/'),
 
+    redirect(r'^support/firefox/?$', 'https://support.mozilla.org/products/firefox'),
```

Comment:
Please bunch these up together since they are all related, and reference the bug numbers so we can look back at how & when they we're added e.g:

```
# bug 682619, 1236910
redirect()
redirect()
etc
```